management_api_app/db/repositories/workspaces.py

```diff
@@ -60,6 +66,12 @@ def create_workspace_item(self, workspace_input: WorkspaceInCreate) -> Workspace
 
         return workspace
 
+    def get_new_address_space(self, cidr_netmask: int = 24):
```

Comment:
Can the 24 be an environment variable? Will be cases in the near future needing larger address spaces for a workspace.

Reply:
We have another task #773 which might give the result you're looking for, but we can also have the default size as an environment variable.
TBH, I don't understand the flow... Is the user expected to know the size required for a workspace?
@marrobi

Reply:
#773 will cover it for now. There are a few different flows, I can forsee. 

1. TRE Admin wants to specify specify address space. (custom) 
2. Address space automatically configured based on size specified by azure admin. (env var? )
3. TRE Admin wants to select from a t-shirt size.

1 and 2 are the ones feel will be most useful initially.

Reply:
Resolving this as it will be handled in another task.